src/main/scala/mesosphere/marathon/raml/PodStatusConversion.scala

```diff
@@ -0,0 +1,116 @@
+package mesosphere.marathon.raml
+
+import mesosphere.marathon.core.instance.{ Instance, InstanceStatus }
+import mesosphere.marathon.core.pod.PodDefinition
+import mesosphere.marathon.state.RunSpec
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable.Seq
+
+trait PodStatusConversion {
+
+  import PodStatusConversion._
+
+  /**
+    * generate a pod instance status RAML for some instance.
+    */
+  @throws[IllegalArgumentException]("if you provide a non-pod `spec`")
```

Comment:
I had a doc comment instead. scaladoc puked. So I got rid of the scaladoc and used this instead. I think it's useful to annotate somehow. Happy to add back as a doc item if scaladoc stops whining and failing the test phase.